src/Components/TrainingListPage.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, Image, TouchableOpacity, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+import theme from '../styles/theme.style';
+
+const getTrainingDuration = trainingDrills => {
+  const durationList = trainingDrills.map(({ durationInMinutes }) => durationInMinutes);
+  const totalDurationInMinutes = durationList.reduce((a, b) => a + b, 0);
+  return totalDurationInMinutes;
+};
+
+const getTrainingNbPlayers = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  const totalNbPlayers = Math.max(...nbPlayersList);
```

Comment:
Here you could return directly the result of Math.max(...) without assigning it to a const